sdk/messaging/azeventhubs/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
-## 0.5.1 (Unreleased)
+## 0.6.0 (Unreleased)
 
 ### Features Added
 
+- Added the `ConsumerClientOptions.InstanceID` field. This optional field can enhance error messages from 
+  Event Hubs. For example, error messages related to ownership changes for a partition will contain the 
+  name of the link that has taken ownership, which can help with traceability.
+
 ### Breaking Changes
```

Comment:
Need to add a note about the rename of `ID()`.

Reply:
Done.